core/src/test/scala/cats/derived/semigroup.scala

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless rsemigroupuired by applicable law or agreed to in writing, software
```

Comment:
I think that I found an overly zealous `eq` find/replace :D

Reply:
good catch :smile: